SU-SWS/ace-casbsgryphon
# CAS-198: tweaks to styling for custom views

# READY FOR REVIEW

# Summary
- Tiny styling tweaks so things look less weird
- Tweak to order of elements so things look less weird

# Review By (Date)
- Probably tomorrow

# Urgency
- Tomorrow

# Steps to Test

1. Go to Recent work or People
2. Notice there isn't a big black line
3. Notice that there is better padding

# Affected Projects or Products
- Recent works
- People

# Associated Issues and/or People
CAS-198

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
